docker/test/integration/steps/steps.py

```diff
@@ -180,6 +248,13 @@ def step_impl(context, file_name, content, path):
     context.test.add_test_data(path, content, file_name)
 
 
+@given("two files with content \"{content_1}\" and \"{content_2}\" are placed in \"{path}\"")
```

Comment:
I am not sure about the purpose of this step. It doesn't seem to be used anywhere and if it could be slightly less specific where it checks an arbitrary number of files?

Reply:
You are right, when I removed the code for the failing scenario, it became unused. Removed it in 4248edb19945b6df8e3bd2b69add3d4ad252d7bc and added to https://issues.apache.org/jira/browse/MINIFICPP-1588 as a possible implementation, which can be changed later.